ccd-definition/ComplexTypes/CourtLocation.json

```diff
@@ -1,18 +1,26 @@
 [
   {
     "ID": "CourtLocation",
-    "ListElementCode": "applicantPreferredCourt",
-    "FieldType": "Text",
+    "ListElementCode": "applicantPreferredCourtLocationList",
```

Comment:
Harry has a point above. ApplicantPreferredCourt is still used in ccd, for instance in RequestedCourt type, applicantRequestedCourt list element code uses it in the element label
The case is load through ccd-data-store, which sends only the fields defined in ccd that the user has the right to read, it is not the whole case, so by removing it we don't have that piece of information.
Maybe just add the ListElementCode back, with a ShowCondition that never evaluates to true, that way civil-service is still the only one writing it but ccd can still read it

Reply:
hey dario, i didn't delete 'applicantPreferredCourt', it's currently on line 13.. git just confusing things